README.md

```diff
@@ -50,3 +51,13 @@ Test your new function; I've provided some basic testing code in `code.test.js`.
 What is the worst-case asymptotic time complexity of your implementation? What
 is the worst-case asymptotic memory complexity? Add your answer, including your
 reasoning, to this markdown file.
+
+**MY ANSWER:**
+
+In terms of how many possible tours there are, we know that there are n! permutations. So then, in the worst case, we will find improvements until all permutations have been considered, in which case we will find improvements no longer. So the outer while loop runs for n! iterations.
+
+We do know that we have 2 nested loops that iterate over all the elements in the currentRoute array. So this part of our code gives us the expression $n^2$.
```

Comment:
Where are these nested loops?

Reply:
There is a for loop inside a for loop inside the outer while loop.

Reply:
Ah ok. But then it seems like you're missing what you're doing for each tour in the analysis.